doc/architecture/reference_points_coordinate_systems.adoc

```diff
@@ -0,0 +1,19 @@
+= Coordinate systems and reference points
+
+OSI uses DIN ISO 8855:2013-11 for coordinate systems and transformations between coordinate systems.
+OSI uses three coordinate systems:
+
+World coordinate system::
+Coordinate system for all entities which are part of ground truth.
+The world coordinate system is an inertial x/y/z-coordinate system.
+The origin is the global reference point, which is determined by the environment simulation.
+This reference point may be derived from map data or other considerations.
+World coordinates can be mapped to a geographic coordinate system via `osi3::GroundTruth::proj_string`.
+
+Sensor coordinate system::
+Coordinate system for all entities which are part of sensor data.
+The origin is the mounting position of the physical sensor or a virtual mounting position, depending on the OSI message.
+
+Object coordinate system::
+Local object coordinate system.
+The origin of the corresponding coordinate systems is not necessarily identical to the center of the object's bounding box.
```

Comment:
Does it then depend on the specific property?

Reply:
@pasched to be done.